src/components/Link/Link.scss

```diff
@@ -5,7 +5,6 @@ $block: '.#{$ns}link-block';
 
 #{$block} {
     $root: &;
-    margin-top: $indentXS;
```

Comment:
Could this change to affect other components which use this Link?

Reply:
Yes. I changed margin in src/blocks/Questions/QuestionBlockItem/QuestionBlockItem.scss. Also Content block uses Link but margins there were already redefined. And that's all appereances I've found

Reply:
If this component used in some project as exported, it can be major-change

Reply:
I restored Link margin-top and margins for related components.